### PR TITLE
profiles: refactor log viewers

### PR DESCRIPTION
**Pull request by glitsj16**
Introduces a common profile for existing GUI log viewer applications.

### CODE REVIEW

File: etc/profile-m-z/profile-m-z/profile-m-z/system-log-common.profile [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+include whitelist-run-common.inc
+include whitelist-runuser-common.inc
+include whitelist-usr-share-common.inc
+include whitelist-var-common.inc
+
+apparmor
+caps.drop all
+ipc-namespace
+net none
+#no3d
```
Comment (kmk3) [inline etc/profile-m-z/profile-m-z/profile-m-z/system-log-common.profile:31]:
```suggestion
no3d
```

3D is unlikely to be needed for logs.

**Review by kmk3 — APPROVED** [review]
Note: It's a bit hard to figure out what are the common lines that are being
moved and what is actually being changed (that is, how this affects the
existing profiles), but at least the new file looks good to me.

I'd suggest trying to split these 2 types of changes in future PRs if possible
to make them easier to review.

**Comment by glitsj16** [comment]
> I'd suggest trying to split these 2 types of changes in future PRs if possible
to make them easier to review.

@kmk3 I'll keep that in mind. Thanks for reviewing, however uneasy!

**Comment by glitsj16** [comment]
@kmk3 Something went very bad with this PR. At first I thought the newly introduced `system-log-common.profile` was missing. But apparently it's placed under `/etc/profile-m-z/profile-m-z/`. Not sure how I did that but it's obviously NOT the correct place to put it. I realize it's my mistake, hence I should fix it. But I don't want to create a bigger mess by randomly fireing git commands I don't feel familiar with in this context. Can you help to get this fixed properly please?

**Comment by kmk3** [comment]
@glitsj16 All good; fixed on commit f5534fb60 ("profiles: fix path of
system-log-common.profile", 2023-09-23).

**Comment by glitsj16** [comment]
@kmk3 Thanks for your kind and knowledgeable work!

**Pull request merged**